### PR TITLE
Dev/create new task functions - Sai bignesh

### PR DESCRIPTION
**Pull request by shivamgoyal7**
**Introduction**
Added create new task functions in worktrolly.

**Descriptions**
To store data in the firestore using cloud functions.

**How to Test**
added proper id in the HTML file.
added function in index.js

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -45,93 +45,91 @@
...
-                    <div class="form-group col-md-4">
-                        <label for="inputState">Status</label>
-                        <input type="text" class="form-control" id="#" list="statusCheck">
-                        <datalist id="statusCheck">
+            </div>
+            <div class="form-row">
+                <div class="form-group col-md-4">
+                    <label for="inputState">Status</label>
+                    <input type="text" class="form-control" id="statusCreateNewTask" list="statusCheck">
+                    <datalist id="statusCheck">
```
Comment (viv9k) [inline public/index.html:123]: Can we make the select option menu as bootstrapped collection

File: public/scripts/worktrollyUI.js [hunk 2]
```diff
@@ -14,22 +14,43 @@ $("#backToMainFromCreateNewTask").click(function() {
...
+    console.log(difficulty);
     console.log(creator);
     console.log(assignee);
+    console.log(estimatedTime);
+    console.log(status);
+    console.log(category);
+
+    var createNewTask = firebase.functions().httpsCallable('createNewTask');
+    createNewTask({ Title: title, Description: des, Priority: priority, Difficulty: difficulty, Creator: creator, Assignee: assignee, EstimatedTime: estimatedTime, Status: status, Category: category }).then(result => {
+        console.log(result.data);
```
Comment (viv9k) [inline public/scripts/worktrollyUI.js:43]: The variable and method name is same, Please use different names

**Comment by viv9k** [comment]
And also resolve conflict

**Comment by shivamgoyal7** [comment]
> And also resolve conflict

resolved merge conflict

File: functions/index.js [hunk 3]
```diff
@@ -1,7 +1,53 @@
-const functions = require('firebase-functions');
-var cors = require('cors')({ origin: true });
 
-const admin = require('firebase-admin');
-admin.initializeApp();
```
Comment (viv9k) [inline functions/index.js:5]: Please check according to the update important lines are removed.

Comment (shivamgoyal7) [reply functions/index.js]: okay

**Pull request merged**